apm-agent-plugins/apm-jaxrs-plugin/src/main/java/co/elastic/apm/agent/jaxrs/JaxRsConfiguration.java

```diff
@@ -39,6 +39,16 @@
         .dynamic(false)
         .buildWithDefault(true);
 
+    private final ConfigurationOption<Boolean> usePathAnnotationValueForTransactionName = ConfigurationOption.booleanOption()
+        .key("use_path_annotation_value_for_transaction_name")
```

Comment:
It probably makes sense to have a single configuration setting which also sets the transaction name for Spring MVC based on the annotations. IIRC, for Spring MVC, there's even a request attribute for that.

Reply:
Where should I move it? to "core configuration options" or "http configuration options"?